egs/tal_csasr/ASR/conv_emformer_transducer_stateless2/train2.py

```diff
@@ -0,0 +1 @@
+../../../librispeech/ASR/conv_emformer_transducer_stateless2/train2.py
```

Comment:
I suggest that you copy train.py to train2.py and change it according to train2.py from `librispeech/ASR/conv_emformer_transducer_stateless2/train2.py`